### PR TITLE
Iss 24 api serializer for classes from core

### LINKED ISSUE #24: api - serializer for classes

### PR DESCRIPTION
**Pull request by Hamilkar247**
fixes #24

### CODE REVIEW

**Comment by Hamilkar247** [comment]
ATTENTION merge after ISS-23 https://github.com/BarskiTeam/BostarDecksWeb/pull/26 this pull

File: core/models.py [hunk 1]
```diff
@@ -6,10 +6,15 @@ class Deck(models.Model):
     description = models.CharField(max_length=1200, verbose_name="Description of deck")
     tag = models.CharField(max_length=50, verbose_name="tag")
     flashCard = models.ManyToManyField('FlashCard', related_name='flashCard', through='DeckFlashCard')
+    public = models.BooleanField(default=False)
+    owner = models.ForeignKey("auth.User", related_name="owner", on_delete=models.CASCADE, null=True)
```
Comment (jlatusek) [inline core/models.py:10]: you push the same changes in ISS 23

**Comment by jlatusek** [comment]
Merge after ISS23 is totally wrong, you supposed to push this 2 commit in one merge request.
After all you have in this merge request the same commit as in ISS23

**Comment by Hamilkar247** [comment]
Ok, ready to review

**Pull request merged**